cli.js

```diff
@@ -1,28 +1,28 @@
 #!/usr/bin/env node
 'use strict';
 global.Promise = require('pinkie-promise');
-var childProcess = require('child_process');
-var meow = require('meow');
-var updateNotifier = require('update-notifier');
-var arrify = require('arrify');
-var globby = require('globby');
-var pkgConf = require('pkg-conf');
-var inquirer = require('inquirer');
-var assign = require('lodash.assign');
-var npmRunPath = require('npm-run-path');
-var isGitClean = require('is-git-clean');
-var utils = require('./cli-utils');
-var codemods = require('./codemods.json');
+const childProcess = require('child_process');
```

Comment:
Change out this with [`execa`](https://github.com/sindresorhus/execa) while you're at it. You can then remove the `npmRunPath` stuff as it's built into `execa`.